### DISCUSSION TITLE
On bookmarks

### DISCUSSION THREAD

**Discussion by XelGia** [sumatrapdfreader/sumatrapdf#2840] [Q&A]
Hi! I noticed that, in v3.4.6, when you collapse the bookmarks and then switch to another tab, any bookmark you've expanded and collapsed opens up again. There is any option you can toggle to prevent this? Thanks!

**Comment by kjk** [comment]
I either don't understand the issue or can't reproduce it. Can you provide more information, preferably with screenshots.

**Reply by GitHubRulesOK** [reply]
@kjk I thing that above description fits with https://github.com/sumatrapdfreader/sumatrapdf/issues/1721

**Reply by XelGia** [reply]
Well, actually yeah. Is the same problem, and is still broken.

**Reply by XelGia** [reply]
To reproduce:
1. Open two doc (doesn't matter if its .pdf or .epub) 
2. Hit a bookmark on one of those (or hit "expand all")
3. Close the bookmark (or hit "collapse all")
4. Switch tabs
5. Switch back again
The tabs stay open, they shouldn't. Only way I could work it out is collapse all bookmarks, THEN close the tab. They stay closed, but is kinda hard when you're opening and closing bookmarks on things like omnibuses and encyclopedias
